posthog/models/plugin.py

```diff
@@ -0,0 +1,23 @@
+from django.contrib.postgres.fields import JSONField
+from django.db import models
+
+
+class Plugin(models.Model):
+    name: models.CharField = models.CharField(max_length=200, null=True, blank=True)
+    description: models.TextField = models.TextField(null=True, blank=True)
+    url: models.CharField = models.CharField(max_length=800, null=True, blank=True)
+    config_schema: JSONField = JSONField(default=dict)
```

Comment:
I think it's worth adding some clarification on what this stores, it can be a bit confusing with the `PluginConfig` below the first time you see it.